perses/data/amino_acid_templates/ALA.pdb

```diff
@@ -0,0 +1,12 @@
+ATOM      1  N   ALA     2      -0.677  -1.230  -0.491  1.00  0.00           N  
```

Comment:
adding `.pdb` templates for all of the amino acids; these are written directly into oemols so that aromaticity, and stereochemistry can be directly perceived by oechem